lib/util/stream_utils.js

```diff
@@ -454,13 +470,15 @@ shaka.util.StreamUtils = class {
               shaka.util.StreamUtils.getCorrectAudioCodecs_(audioCodecs);
           const audioFullType = shaka.util.MimeUtils.getFullOrConvertedType(
               video.mimeType, audioCodecs, ContentType.AUDIO);
-          if (!MediaSource.isTypeSupported(audioFullType)) {
+          if (!shaka.util.StreamUtils
```

Comment:
If you define a local constant for "StreamUtils" (or "MediaSourceEngine", as I suggest in another comment), at the start of this function, you could fit this in a single line:
```js
          if (!StreamUtils.getMediaSourceSupportedType(audioFullType)) {
```

Reply:
When I tried to replace it in MediaSourceEngine, I got `ERROR - [JSC_MISSING_REQUIRE_IN_PROVIDES_FILE] 'shaka.media.MediaSourceEngine' references a namespace which was not required by this file. `. Maybe it can not be done so by constructor `MediaSourceEngine`